### PR TITLE
Update install.rst

### PR DESCRIPTION
**Pull request by kegl**
Adding necessary command to run tests.

ramp-database and ramp-utils are fine now, engine and frontend still fail. It would be nice to fix this.

### CODE REVIEW

**Comment by kegl** [comment]
Actually ramp-engine is fine if I pip install it in development mode.

File: doc/install.rst [hunk 1]
```diff
@@ -83,7 +83,17 @@ Test
 ====
 
 If you are developing for the RAMP package, you will be interested about
-testing your new feature. You can run the test using ``pytest`` from the root
+testing your new feature. First you need to `set up the database <https://paris-saclay-cds.github.io/ramp-docs/stable/setup_server.html#set-up-the-ramp-database>`_ and create the test database::
```
Comment (agramfort) [inline doc/install.rst]: this is not the right way to link within sphinx. you need to link to local build.

Comment (agramfort) [reply doc/install.rst]: a dev build of the website would point to stable

**Comment by glemaitre** [comment]
I don't think that this test section should be placed in the `install.rst`. After some feedback from @lucyleeow and @maikia, it seems that this section should be in a `maintainer.rst` or something similar. The reason is that we are expecting specific username and password to test on the database and people get confused with the part of the deployment then.

File: doc/install.rst [hunk 2]
```diff
@@ -83,7 +83,17 @@ Test
...
+
+    createdb --owner=<the owner you specified> databoard_test
```
Comment (glemaitre) [inline doc/install.rst]: Here you will not have any choice. The config that we are testing thing require `mrramp` as password and user.

**Pull request closed without merging**